assets/styles/components/_tab-toggle.scss

```diff
@@ -1,34 +1,300 @@
+    border-top-left-radius: 0;
+    border-top-right-radius: 0;
+    border-bottom-left-radius: 4px;
+    border-bottom-right-radius: 4px;
+
+    dl {
+      background-color: transparent;
+    }
+
+    h2 > a,
+    h3 > a,
+    h4 > a,
+    h5 > a {
+      color: #000;
+      border-bottom: 1px solid transparent;
+      &:hover {
+        color: #000;
+        border-bottom: 1px solid #000;
+      }
+    }
+
+    .text-center img {
+      border: 1px solid $ddgray;
+      border-radius: 8px;
+    }
+
+    .card-body img {
+      border: 0px solid $ddgray;
+    }
+  }
+
+  // Standard tab layout
+  .nav-tabs {
+    list-style-type: none;
+    padding: 0;
+    display: flex;
+    flex-wrap: wrap;
+    gap: 0;
+    margin: 0;
+    position: relative;
+    z-index: 2;
+    overflow-x: auto;
+    -webkit-overflow-scrolling: touch;
+    scrollbar-width: thin;
+    scrollbar-color: rgba(0, 0, 0, 0.2) transparent;
+    border-bottom: none;
+    padding-left: 0 !important;
+    overflow-y: hidden;
+
+    &::-webkit-scrollbar {
+      height: 4px;
+    }
+
+    &::-webkit-scrollbar-track {
+      background: transparent;
+    }
+
+    &::-webkit-scrollbar-thumb {
+      background: rgba(0, 0, 0, 0.2);
+      border-radius: 4px;
+    }
+
+    li {
+      padding: 0;
+      position: relative;
+      flex: 0 0 auto;
+      white-space: nowrap;
+      box-sizing: border-box;
+
+      &:before {
+        content: '' !important;
+        margin: 0 !important;
+        float: none !important;
+      }
+
+      &:lang(ja) {
+        font-style: normal;
+        font-weight: bold;
+        font-size: 14px;
+        line-height: 24px;
+      }
+
+      &.active {
+        z-index: 3;
+
+        a {
+          @extend %active-tab-base;
+          transition: none;
+          box-sizing: border-box;
+
+          &:after {
+            content: '';
+            position: absolute;
+            top: -1px;
+            left: -1px;
+            width: calc(100% + 2px);
+            height: 3px;
+            background: linear-gradient(90deg, $tab-active-color, lighten($tab-active-color, 20%));
+            border-radius: 3px 3px 0 0;
+          }
+
+          &:hover {
+            color: $tab-active-color;
+            transition: none;
+          }
+
+          &::before {
+            display: none;
+          }
+        }
+      }
+
+      &:not(.active) {
+        a {
+          transition: none;
+          box-sizing: border-box;
+
+          &:hover {
+            color: $tab-active-color;
+            background: $tab-hover-bg;
+            border-color: transparent;
+            border-top-left-radius: 4px;
+            border-top-right-radius: 4px;
+            transition: color 0.2s ease, background-color 0.2s ease;
+          }
         }
+      }
 
-        &:hover {
-            box-shadow: 2px 5px 12px rgba(0, 0, 0, 0.1);
-            border: none;
+      a {
+        @extend %tab-link-base;
+        box-sizing: border-box;
+
+        &::before {
```

Comment:
100% optional, but it might be worth adding a comment on top of more abstract blocks like these, just to give a sense of what's being drawn.